fhir-config/src/main/java/com/ibm/fhir/config/FHIRConfiguration.java

```diff
@@ -25,6 +25,7 @@ public class FHIRConfiguration {
     // Core server properties
     public static final String PROPERTY_ORIGINAL_REQUEST_URI_HEADER_NAME = "fhirServer/core/originalRequestUriHeaderName";
     public static final String PROPERTY_TENANT_ID_HEADER_NAME = "fhirServer/core/tenantIdHeaderName";
+    public static final String PROPERTY_SHARD_KEY_HEADER_NAME = "fhirServer/core/shardKeyHeaderName";
     public static final String PROPERTY_DATASTORE_ID_HEADER_NAME = "fhirServer/core/datastoreIdHeaderName";
```

Comment:
seeing these next to each other makes me wonder whether/how they relate.
datastore_id was our previous mechanism for supporting multiple datastores for a single tenant.
does it make sense to *retire* that in favor of this new sharding strategy?

Reply:
They can live with each other. Sharding happens within a particular database. You may want to separate data across different datastores for other functional/security reasons.